server/test/handlers/inputSchedule.test.js

```diff
@@ -1,22 +1,39 @@
-const timeslotHandler = require('../../models/handlers/timeslot');
+const timeslotHandler = require('../../models/handlers/workerTimeslot');
```

Comment:
Hi Peter,
Similarly to `routes/private.js`, I would name the handler to `workerTimeslotHandler` instead of timeslotHandler to avoid later confusion.

Reply:
completed!